source/MaterialXRenderGlsl/TextureBaker.cpp

```diff
@@ -162,9 +194,6 @@ void TextureBaker::bakeGraphOutput(OutputPtr output, GenContext& context, const
 
     renderTextureSpace();
 
-    // TODO: Add support for graphs containing geometric nodes such as position and normal.
```

Comment:
While we're getting closer to support for "position" and "normal" nodes in pattern graphs, we're not quite there yet, so we should hold onto this comment.  :)